content/integrate/prometheus-with-redis-enterprise/prometheus-metrics-definitions.md

```diff
@@ -20,262 +20,261 @@ Here are the metrics available to Prometheus:
+| <span class="break-all">redis_connected_slaves</span> | <span class="break-all">`redis_server_connected_slaves`</span> | Number of connected replicas |
+| <span class="break-all">redis_db0_avg_ttl</span> | <span class="break-all">`redis_server_db0_avg_ttl`</span> | Average TTL of all volatile keys |
+| <span class="break-all">redis_db0_expires</span> | <span class="break-all">`redis_server_expired_keys`</span> | Total count of volatile keys |
+| <span class="break-all">redis_db0_keys</span> | <span class="break-all">`redis_server_db0_keys`</span> | Total key count |
+| <span class="break-all">redis_evicted_keys</span> | <span class="break-all">`redis_server_evicted_keys`</span> | Keys evicted so far (since restart) |
+| <span class="break-all">redis_expire_cycle_cpu_milliseconds</span> | <span class="break-all">`redis_server_expire_cycle_cpu_milliseconds`</span> | The cumulative amount of time spent on active expiry cycles |
+| <span class="break-all">redis_expired_keys</span> | <span class="break-all">`redis_server_expired_keys`</span> | Keys expired so far (since restart) |
+| <span class="break-all">redis_forwarding_state</span> | <span class="break-all">`redis_server_forwarding_state`</span> | Shard forwarding state (on or off) |
+| <span class="break-all">redis_keys_trimmed</span> | <span class="break-all">`redis_server_keys_trimmed`</span> | The number of keys that were trimmed in the current or last resharding process |
+| <span class="break-all">redis_keyspace_read_hits</span> | <span class="break-all">`redis_server_keyspace_read_hits`</span> | Number of read operations accessing an existing keyspace |
+| <span class="break-all">redis_keyspace_read_misses</span> | <span class="break-all">`redis_server_keyspace_read_misses`</span> | Number of read operations accessing a non-existing keyspace |
+| <span class="break-all">redis_keyspace_write_hits</span> | <span class="break-all">`redis_server_keyspace_write_hits`</span> | Number of write operations accessing an existing keyspace |
+| <span class="break-all">redis_keyspace_write_misses</span> | <span class="break-all">`redis_server_keyspace_write_misses`</span> | Number of write operations accessing a non-existing keyspace |
+| <span class="break-all">redis_master_link_status</span> | <span class="break-all">`redis_server_master_link_status`</span> | Indicates if the replica is connected to its master |
+| <span class="break-all">redis_master_repl_offset</span> | <span class="break-all">`redis_server_master_repl_offset`</span> | Number of bytes sent to replicas by the shard; calculate the throughput for a time period by comparing the value at different times |
+| <span class="break-all">redis_master_sync_in_progress</span> | <span class="break-all">`redis_server_master_sync_in_progress`</span> | The master shard is synchronizing (1 true | 0 false) |
```

Comment:
Looks like the vertical bar here creates an extra invisible column, so you can't see the "0 false" part.
```suggestion
| <span class="break-all">redis_master_sync_in_progress</span> | <span class="break-all">`redis_server_master_sync_in_progress`</span> | The master shard is synchronizing (1 true; 0 false) |
```